gensim/models/fasttext.py

```diff
@@ -27,10 +30,87 @@
 
 .. sourcecode:: pycon
 
-    >>> from gensim.test.utils import common_texts
-    >>> from gensim.models import FastText
+    >>> # from gensim.models import FastText  # FIXME: why does Sphinx dislike this import?
```

Comment:
what's a problem with it, can you explain, please?

Reply:
aha, instead a current comment please add `# noqa: F401` (this will suppress a message)

Reply:
Unfortunately, that does not help. flake8 still errors out with the same message. It seems to be ignoring the noqa directive in the comments.